src/index.js

```diff
@@ -21,6 +21,7 @@ export { default as Resource } from './components/Resource.vue'
 export { default as Spinner } from './components/Spinner.vue'
 export { default as SuccessMessage } from './components/SuccessMessage.vue'
 export { default as TextEditor } from './components/TextEditor'
+export { default as ListManager } from './components/ListManager'
```

Comment:
Can't this simply be:

```suggestion
export ListManager from './components/ListManager'
```

without the need to export `default` explicitly in the index file?

Reply:
@Alchez this is done just to make all the exports consistent